lib/kb_orthofinder/kb_orthofinderImpl.py

```diff
@@ -277,31 +278,41 @@ def annotate_plant_transcripts(self, ctx, input):
         if(len(sequences_dict)==0):
             raise Exception("The genome does not contain any protein sequences!")
 
+        # We have a problem with how OrthoFinder arbitrarily replaces 'special' characters in identifiers:
+        # https://github.com/davidemms/OrthoFinder/blob/master/scripts_of/util.py#L181
+        # without 'hacking' the OF code, here I'm trying to anticipate how OF may have changed the identifier
+
+        alt_seq_ids_dict=dict()
```

Comment:
just using 
```
alt_seq_ids_dict = {}
```
is shorter and more performant

Reply:
I never really got the hang of Python one-liners, but I decided a while ago that `dict()` and `list()` were easier for me to read than `{}` and `()`

Reply:
OTOH, if you use shorthands you will get familiar with them pretty quickly and you'll recognise them in other people's code and not have to look up what they mean.